tls_codec/Cargo.toml

```diff
@@ -11,7 +11,7 @@ edition = "2021"
 rust-version = "1.56"
 
 [dependencies]
-zeroize = { version = "1", default-features = false, features = ["alloc", "zeroize_derive"] }
+zeroize = { version = "1", default-features = false, features = ["alloc"] }
```

Comment:
IIUC the `zeroize_derive` feature is not used in any way by this crate, so I removed it.